src/mono/wasm/build/WasmApp.targets

```diff
@@ -85,6 +85,8 @@
                                               - AppBundle directly contains user files
                                               - AppBundle/_framework contains generated files (dlls, runtime scripts, icu)
                                               - AppBundle/_content contains web files from nuget packages (css, js, etc)
+      - $(WASMStripIL)                      - Set to true to enable trimming away AOT compiled methods body (IL code)
```

Comment:
nit: all the other properties are named `Wasm` not `WASM`

Reply:
Good catch. Updated.